res/values-ja-rJP/strings.xml

```diff
@@ -16,9 +16,9 @@
   <string-array name="actionbar_sortby">
     <item>A-Z</item>
     <item>最新 - 最古</item>
+    <!--TODO re-enable when server-side folder size calculation is available   
+    <item>Biggest - Smallest</item>-->
```

Comment:
Please do not modify any language specific files as they are all handled by Transifex.